application/common/models/Mfa.php

```diff
@@ -493,6 +509,7 @@ public static function getTypes()
             self::TYPE_MANAGER => 'Manager Backup Code',
             self::TYPE_TOTP => 'Authenticator App',
             self::TYPE_WEBAUTHN => 'Security Key',
+            self::TYPE_RECOVERY => 'Recovery Backup Code',
```

Comment:
It might be better to call this a "Recovery Contact Code", since it's for use by a recovery contact to help the person recover from having lost their MFA methods:
```suggestion
            self::TYPE_RECOVERY => 'Recovery Contact Code',
```

I think we used "Backup" in the manager option to communicate that concept of helping someone get back into their account, but "recovery backup" doesn't quite communicate our intended purpose, I think.

Feel free to speak up, if you see it differently.